### ISSUE TITLE
1-function typo

### ISSUE THREAD

**Issue by kyle-messier** [wlandau/targets-tutorial#17]
## Description

Simple typo in the Rmarkdown for tutorial 1-functions.Rmd

L78, the training and test set sizes are reversed in the plain language text. 

## Reproducible example

L78 says: 

The training set has 2113 customers (rows) and the testing set has 4930.

According to the output of the code block below, it says:

<Training/Testing/Total>
<4930/2113/7043>

## Expected result

L78 should say: 

The training set has 4930 customers (rows) and the testing set has 2113.
